src/content/app-platform/create-catalog/index.md

```diff
@@ -117,7 +118,8 @@ kubectl gs template catalog \
   --namespace org-example \
   --description "An example Catalog" \
   --url https://example.github.io/example-catalog/ \
-  --logo https://example.com/logos/example-logo.png
+  --logo https://example.com/logos/example-logo.png \
+  --visibility public
```

Comment:
Does it make sense to add this? As it's the default

Reply:
@MarcelMue Damn I overthought this. My idea was to show you can change it but its already covered in the text.

I've removed. PTAL